### PR TITLE
Covered new releases of ember-collection and ember-keyboard

### PR DESCRIPTION
**Pull request by ijlee2**

### CODE REVIEW

File: source/2020-06-12-the-ember-times-issue-152.md [hunk 1]
```diff
@@ -9,31 +9,45 @@ responsive: true
...
+## [Meet long-awaited `ember-collection` v1.0 release candidate 🎉](https://twitter.com/Alonski/status/1264930582758412288)
 
-<change section title emoji>
-<consider adding some bold to your paragraph>
-<please include link to external article/repo/etc in paragraph / body text, not just header title above>
+After two years, a new version of [`ember-collection`](https://github.com/adopted-ember-addons/ember-collection) is here! You can use this addon to **incrementally render a large list of items**. The official `v1.0` will be released soon.
 
-<add your name to author list, top and bottom>
-<add blurb and emoji to "SOME-INTRO-HERE">
+Two most notable [changes](https://github.com/adopted-ember-addons/ember-collection/releases/tag/v1.0.0-rc.0) are:
```
Comment (jaredgalanis) [inline source/2020-06-12-the-ember-times-issue-152.md]:
```suggestion
Two of the most notable [changes](https://github.com/adopted-ember-addons/ember-collection/releases/tag/v1.0.0-rc.0) are:
```

File: source/2020-06-12-the-ember-times-issue-152.md [hunk 2]
```diff
@@ -9,31 +9,45 @@ responsive: true
...
+
+- Ember upgrade from 2.18 to 3.12!
+- Improved continuous integration and auto-release process
+
+Please thank [Alon Bukai (@Alonski)](https://github.com/Alonski), [Gaurav Munjal (@Gaurav0)](https://github.com/Gaurav0), [Robert Wagner (@rwwagner90)](https://github.com/rwwagner90), and [Vasanth (@vasind)](https://github.com/vasind) for their help with maintaining [`ember-collection`](https://github.com/adopted-ember-addons/ember-collection). If you currently use the addon, we ask that you upgrade and check that your app and tests continue to pass.
 
 ---
 
-## [Section title in sentence case 🐹](section-url)
+## [Use modern keyboard API with `ember-keyboard` v6.0 ⌨️](https://twitter.com/lukemelia/status/1270672606220279808)
```
Comment (jaredgalanis) [inline source/2020-06-12-the-ember-times-issue-152.md]:
```suggestion
## [Use a modern keyboard API with `ember-keyboard` v6.0 ⌨️](https://twitter.com/lukemelia/status/1270672606220279808)
```

File: source/2020-06-12-the-ember-times-issue-152.md [hunk 3]
```diff
@@ -9,31 +9,45 @@ responsive: true
 <SAYING-HELLO-IN-YOUR-FAVORITE-LANGUAGE> Emberistas! 🐹
 
 Learn how to make your existing app accessible 💪,
+Meet long-awaited `ember-collection` v1.0 release candidate 🎉,
+Use modern keyboard API with `ember-keyboard` v6.0 ⌨️,
```
Comment (jaredgalanis) [inline source/2020-06-12-the-ember-times-issue-152.md]:
```suggestion
Use a modern keyboard API with `ember-keyboard` v6.0 ⌨️,
```

File: source/2020-06-12-the-ember-times-issue-152.md [hunk 4]
```diff
@@ -9,31 +9,45 @@ responsive: true
 <SAYING-HELLO-IN-YOUR-FAVORITE-LANGUAGE> Emberistas! 🐹
 
 Learn how to make your existing app accessible 💪,
+Meet long-awaited `ember-collection` v1.0 release candidate 🎉,
```
Comment (jaredgalanis) [inline source/2020-06-12-the-ember-times-issue-152.md]:
```suggestion
Meet the long-awaited `ember-collection` v1.0 release candidate 🎉,
```

File: source/2020-06-12-the-ember-times-issue-152.md [hunk 5]
```diff
@@ -9,31 +9,45 @@ responsive: true
...
+Use modern keyboard API with `ember-keyboard` v6.0 ⌨️,
 <SOME-INTRO-HERE-TO-KEEP-THEM-SUBSCRIBERS-READING>
 
 READMORE
 
 ---
 
-## [Section title in sentence case 🐹](section-url)
+## [Meet long-awaited `ember-collection` v1.0 release candidate 🎉](https://twitter.com/Alonski/status/1264930582758412288)
```
Comment (jaredgalanis) [inline source/2020-06-12-the-ember-times-issue-152.md]:
```suggestion
## [Meet the long-awaited `ember-collection` v1.0 release candidate 🎉](https://twitter.com/Alonski/status/1264930582758412288)
```

**Review by jaredgalanis — APPROVED** [review]
Looks ✨ ✨ ✨  as usual. Just a few suggestions for you to consider. Nothing really needs to change.

**Comment by ijlee2** [comment]
Thanks, @jaredgalanis! I liked the suggestions that you made and applied all.

**Comment by amyrlam** [comment]
thank you!! i'm going to merge this in and fix upstream

**Pull request merged**